src/web/make-report.rkt

```diff
@@ -100,7 +100,7 @@
      (body
       (header
        (h1 ,(if note (string-titlecase note) "") " Results")
-       (img ([src "logo-car.png"]))
+       (img ([src "./demo/logo-car.png"]))
```

Comment:
Ok, so this is clever but not the right fix. Here's what happens. On the nightly, we don't actually run a single report. We run a bunch of reports (one per file/directory in `bench/`) and then *merge* them into a single report.

So what's going on here is that we copy the `logo-car.png` file into each individual report, but not into the merged report. That happens in `infra/merge.rkt`, at the bottom of the file. You should add `logo-car.png` there and then not have this change.